velox/exec/HashProbe.cpp

```diff
@@ -581,6 +581,17 @@ void HashProbe::addInput(RowVectorPtr input) {
   }
   activeRows_ = nonNullInputRows_;
 
+  // Update statistics for null keys in join operator.
+  // Updating here means we will report 0 null keys when build side is empty.
```

Comment:
Let's update PR description to clarify that HashProbe doesn't populate numNullKeys when build side is empty. Let's also mention this in the comment for numNullKeys stat.